src/main/java/Task.java

```diff
@@ -0,0 +1,38 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    private static int count = 0; //Class level attribute, number of tasks
+    private static Task[] list = new Task[100];
```

Comment:
maybe a constant could be defined for this to avoid magic integers to improve the readers understanding of why the number 100 was used